Tests/MeiliSearchUnitTests/MockURLSession.swift

```diff
@@ -59,6 +60,7 @@ class MockURLSession: URLSessionProtocol {
 }
 
 class MockURLSessionDataTask: URLSessionDataTaskProtocol {
+  var request: URLRequest?
```

Comment:
Does it need to be mutable?

Reply:
I think so, because in the line 23 I'm mutating it to receive the value of the current request :)